compiler/native/environment_test.go

```diff
@@ -6,6 +6,7 @@ package native
 
 import (
 	"flag"
+	"github.com/urfave/cli/v2"
```

Comment:
nit: revert import order change.

Reply:
yea, we should configure one of the linters to catch this for us. i think we have discrepancies on import order in a few places.